src/styles/blocks/body.scss

```diff
@@ -0,0 +1,13 @@
+body {
```

Comment:
body is not a block so move it inside index.scss or create file in different folder with name like main, global itp